### PR TITLE
Incorporate feedback from run-through with new developer

### PR DESCRIPTION
**Pull request by eecavanna**
⚠️ These are "quick 'n dirty" notes written by someone that may have a fever (he can't find his thermometer).

### CODE REVIEW

**Comment by eecavanna** [comment]
Hi @pkalita-lbl, I don't expect you to merge these into the branch you are working on. This was the most "standard" way I thought of to convey my annotations to you. You can use them however you want.

File: docs/development.md [hunk 1]
```diff
@@ -133,9 +146,18 @@ Although the project is designed to be run in Docker, having the dependencies in
...
+<!-- Note: Add `--build` to this command so that Docker Compose builds
+           the containers, rather than pulling from from GHCR (unless you
+           want to use the versions that happen to currently be on GHCR).
+           This has to do with the fact that the `docker-compose.yml` file 
+           contains service specs having both an `image` and `build` section. -->
+
 ```bash
 docker compose up -d
 ```
+> The `-d` is short for `--detach` and makes it so the container logs _don't_ take over your shell, causing you to have to open up a new shell in order to run more commands.
```
Comment (eecavanna) [inline docs/development.md:158]:
```suggestion
> The `-d` is short for `--detach` and makes it so the container logs (STDOUT and STDERR streams) _don't_ take over your shell, causing you to have to open up a new shell in order to run more commands.
```

File: docs/development.md [hunk 2]
```diff
@@ -133,9 +146,18 @@ Although the project is designed to be run in Docker, having the dependencies in
...
+
+Troubleshooting: If the building of one of the service fails due to networking timeouts, but the building of all other services completes successfully, we recommend you retry building that service alone. Our thinking is that there will be less demand on your network that way. You can do that via `$ docker compose build {service_name}` (e.g. `$ docker compose build web`).
```
Comment (eecavanna) [inline docs/development.md:160]:
```suggestion
Troubleshooting: If the building of one of the services fails with an error citing networking timeouts, _but_ the building of _other_ services completes successfully, we recommend you retry building only the service that failed, by itself. Our thinking is that there will be less demand on your network that way. You can do that via `$ docker compose build {service_name}` (e.g. `$ docker compose build web`).
```

**Comment by eecavanna** [comment]
You can close this PR once you determine it has served its purpose.

**Comment by pkalita-lbl** [comment]
Leaving a note to self that I also want to add a section on how to load your local database without NERSC access. The first step will be "talk to someone who can get a backup file for you", but it's better than nothing.

**Review by pkalita-lbl — APPROVED** [review]

**Pull request merged**